src/main/java/duke/command/Parser.java

```diff
@@ -0,0 +1,94 @@
+package duke.command;
```

Comment:
Parser.java seems to be not part of the commands, might be better if you put into another directory?